app/main/views/g_cloud.py

```diff
@@ -544,6 +544,61 @@ def end_search(framework_framework, project_id):
     )
 
 
+@direct_award.route(
+    '/<string:framework_framework>/projects/<int:project_id>/did-you-award-contract',
+    methods=['GET', 'POST']
+)
+def did_you_award_contract(framework_framework, project_id):
+    all_frameworks = data_api_client.find_frameworks().get('frameworks')
+    framework = framework_helpers.get_latest_live_framework(all_frameworks, framework_framework)
+
+    # Get the requested Direct Award Project.
+    project = data_api_client.get_direct_award_project(project_id=project_id)['project']
+
+    form = DidYouAwardAContractForm()
+    form.did_you_award_a_contract.options = [{"label": label, "value": value}
```

Comment:
Dynamically editing the form object like this should be done in the `__init__` method of the form. The `.options` (or historically `.toolkit_macro_options`) attribute is something we're going to need every time we instantiate this form